x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/details/helpers.ts

```diff
@@ -188,11 +188,11 @@ export const getHostEndpoint = async (
     return null;
   }
 
-  const { esClient, endpointContext, request } = deps;
+  const { esClient, endpointContext } = deps;
   const logger = endpointContext.logFactory.get('metadata');
 
   try {
-    const fleetServices = endpointContext.service.getScopedFleetServices(request);
+    const fleetServices = endpointContext.service.getInternalFleetServices();
```

Comment:
Just to be sure, is this safer? I mean, is this `fleetServicesFactory.asInternalUser();` a safer role to be used in this context? Just asking because my lack of knowledge here

Reply:
Here, its safe - and a good question to ask/scrutinize anytime we see "asInternalUser()".
Its safe here because we're only providing supplemental data to the Host Details in security solution. A few releases back, it was requested that we allow endpoint data to be visible in that page (and Alerts).